docs/local-development.md

```diff
@@ -549,9 +550,52 @@ When we started the Debugger, we were not sure if we would keep webpack and the
 
 Now that we *have* decided that working in github with webpack has a lot of benefits we could switch to JSX. We are open to switching if someone could help us do it, join the [discussion here](https://github.com/devtools-html/debugger.html/issues/1747).
 
+
+### Ninjas
```

Comment:
In general good, not sure about the Ninja part.  I like that its not `maintainers` which always sounds pretty lame.  Keep it fun!  Maybe lets keep asking around for more ideas.